### DISCUSSION TITLE
The --hooksWatch Option Does Not Automatically Restart the Application

### DISCUSSION THREAD

**Discussion by bridevmx** [pocketbase/pocketbase#4222] [Q&A]
I am using Pocketbase to develop my application and I have noticed that the --hooksWatch option is not working as expected. According to the documentation, this option should automatically restart the application when a change is detected in the pb_hooks files. I am using version 20.7.

**Comment by ganigeorgiev** [comment]
It works fine for me, but with 2 caveats:
- currently it is supported only on Linux/Mac (_for Windows there should be a console message asking the user to restart the app manually_)
- the files watcher will be initialized only if the `pb_hooks` directory was created before the app startup

**Reply by bridevmx** [reply]
Understood, it does show the message on the console, but since I am using Windows, that's why it doesn't restart automatically. I had not taken into account that it was not compatible with Windows, so I will restart it manually. Thank you for the response.
